### ISSUE TITLE
Inflector feature: acronyms

### ISSUE THREAD

**Issue by ksenia-zalesnaya** [hanami/utils#154]
In Rails I had model FAQ in `app/models/faq.rb`. It was possible with `ActiveSupport::Inflector.inflections { |inflect| inflect.acronym 'FAQ' }`. 
Hanami expects only `Faq` model and I cannot find settings to change that.  When I rename class to `FAQ` I get an error. Since `FAQ` is acronym I do not want to write it as `Faq`, but I have to.
So I think inflector should capitilize/camelcase `faq` like `FAQ`, not `Faq`.

**Comment by joneslee85** [comment]
Thanks for submitting this issue. I acknowledge this is a useful feature for Inflector, the question is is it needed by everyone on common basis? Would you be able to provide your usecase and why you need this feature with Hanami now?

**Comment by ksenia-zalesnaya** [comment]
Well, maybe FAQ is not so popular example. But a lot of people use API, and it also should be camelized not like Api. Someone may use SMTP, CSRF, CSP, etc.

**Comment by cllns** [comment]
I think we should allow developers to define any acronym inflector they want. (I do not think we should maintain a list of any acronyms, since that'd be a ton of work and confusing. I don't think you were proposing this, but I want to clarify.)

@ksenia-zalesnaya Your use-case was loading classes right?

**Comment by ksenia-zalesnaya** [comment]
@cllns Yes, I meant there should be an opportunity to define acronyms. They should be used in loading classes.

**Comment by cllns** [comment]
@ksenia-zalesnaya any chance you could provide a failing test for that? I just generated a model named `faq`, which got generated at `Faq`. I changed it to `FAQ` in both the entity and its spec and it worked fine. I forget the exact circumstance where you discovered this.

**Comment by ksenia-zalesnaya** [comment]
@cllns I got this issue some time ago and now I also can't make an example with model. But I did it with routes and controller: https://github.com/ksenia-zalesnaya/hanami/commit/15b94d09056ed48ae4cb41840425d0910ef51ab3

**Comment by jodosha** [comment]
@ksenia-zalesnaya Hi! 😄 

What do y'all think about this Ruby basic solution?

```ruby
module Faq
  class Index
    include Hanami::Action

    def call(params)
      self.body = 'hello from Faq'
    end
  end
end

FAQ = Faq
```

**Comment by AlfonsoUceda** [comment]
ping @ksenia-zalesnaya

**Comment by ksenia-zalesnaya** [comment]
@jodosha I thought there could be more elegant solution. 
I do not need acronyms that much in controllers, I need them more in models. Earlier each model should be required individually. Now `Hanami::Utils.require!` is introduced. I wanted to test it to check if it will understand FAQ model, but `hanami new` generates broken application.

**Comment by jodosha** [comment]
@ksenia-zalesnaya 
> but `hanami new` generates broken application.

What do you mean?

**Comment by ksenia-zalesnaya** [comment]
@jodosha When I wrote this I got an error with newly generated app, but later I found I had some mess with installed hanami gems. After I cleaned up and used latest versions I could generate working application.

I tried to create model FAQ and it worked. As I see, `Hanami::Utils.require!` loads constants no matter what the filename is? In Rails I should name file user.rb for User constant, but here it seems to work anyway. It seems I got all I needed. Closing this.
